### PR TITLE
Added corsheaders configuration to settings.py

### PR DESCRIPTION
**Pull request by Ruaaann**
**_WHAT DOES THIS PR DO?_**

Very simple refactoring PR that adds the configuration necessary to enable the django cors-headers app in order to be used in our project. 

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- #31 

**_HOW DO I TEST OUT THIS PR?_**

  - Currently only localhost:8000 is an allowed origin.

### CODE REVIEW

File: api/backend/settings.py [hunk 1]
```diff
@@ -38,11 +38,18 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'rest_framework', # Django REST Framwork
+    'corsheaders',
+]
+
+CORS_ALLOWED_ORIGINS = [
```
Comment (danielogen) [inline api/backend/settings.py:44]: This is great since our backend API is running on PORT 8000

**Review by danielogen — APPROVED** [review]
Hi @Ruaaann this well done!

**Review by azboss2021 — APPROVED** [review]
Looks good @Ruaaann This is good to avoid CORS headers when interacting with the API from frontend

**Review by danielogen — CHANGES_REQUESTED** [review]
Looks like we have some merge conflicts

**Comment by Ruaaann** [comment]
I have addressed the merge conflicts and fixed them, this PR can be merged upon re-review!

**Review by danielogen — APPROVED** [review]
All conflict have been removed. Thanks @Ruaaann

**Pull request merged**